core_v2/sources/domains.move

```diff
@@ -344,9 +344,10 @@ module aptos_names_v2::domains {
     fun register_domain_generic(
         sign: &signer,
         domain_name: String,
-        num_years: u8
+        registration_duration_secs: u64,
     ) acquires CollectionCapabilityV2, NameRecordV2, RegisterNameEventsV1, ReverseRecord, SetNameAddressEventsV1, SetReverseLookupEventsV1 {
         assert!(config::is_enabled(), error::unavailable(ENOT_ENABLED));
+        let num_years = (time_helper::seconds_to_years(registration_duration_secs) as u8);
         assert!(
             num_years > 0 && num_years <= config::max_number_of_years_registered(),
```

Comment:
should we also change `max_number_of_years_registered` to second?

Reply:
ok price is still based on year, do we want to change that too?

Reply:
I was thinking but not sure which one is better

Reply:
I wouldn't want to change the price model while migration, that's why I didn't touch that part. Although we change the interface here, but we probably want to keep the current price for a while?